### ISSUE TITLE
Failed to generate favicons: Could not insert HTML code..

### ISSUE THREAD

**Issue by laggykiller** [jlesage/docker-baseimage-gui#150]
```
RUN \
    APP_ICON_URL=https://github.com/jlesage/docker-templates/raw/master/jlesage/images/generic-app-icon.png && \
    install_app_icon.sh "$APP_ICON_URL"
```

Leads to `Failed to generate favicons: Could not insert HTML code..`.

API change? Also consider supporting offline favicon generation such as using ImageMagick?

**Comment by jlesage** [comment]
I'm looking at this. It seems that the `versioning` parameter of the API doesn't work anymore.

**Comment by jlesage** [comment]
Fixed with version 4.6.6.
